### ISSUE TITLE
Cipher Group Support

### ISSUE THREAD

**Issue by ab-gmbh** [F5Networks/f5-appsvcs-extension#128]
##### Do you already have an issue opened with F5 support?
Github Issues are consistently monitored by F5 staff, but should be considered as best effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open an case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

##### ISSUE TYPE

 - Feature Idea

##### AS3 BUILD/ VERSION

```
3.13.1 LTS release
```

##### BIGIP VERSION

```

```

##### SUMMARY

```
Currently there is no Cipher Group support in AS3, raising this as an enhancement request. 
```

##### STEPS TO REPRODUCE

```

```

##### EXPECTED RESULTS

```

```

##### ACTUAL RESULTS

```

```

**Comment by ccchub** [comment]
Correct

**Comment by dstokesf5** [comment]
Thank you for your input, I have added this to our internal backlog as AT-691.

**Comment by dstokesf5** [comment]
This should be resolved in the 3.17.0 release. If you are still experiencing a problem, please re-open this issue or create a new one.
